zeebe.js

```diff
@@ -0,0 +1,96 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+const authorizationConfiguration = {
+  clientId: process.env.ZEEBE_CLIENT_ID,
+  clientSecret: process.env.ZEEBE_CLIENT_SECRET,
+  audience: process.env.ZEEBE_AUDIENCE
+};
+
+// An action that assigns a user task.
+async function assignUser([assignee]) {
+    // Every request needs an access token.
+    const accessToken = await getAccessToken(authorizationConfiguration);
+  
+    // These settings come from your .env file.
+    const ZeebeApiUrl = process.env.ZEEBE_BASE_URL;
+    const userTaskKey = process.env.USER_TASK_KEY;
+  
+    // This is the API endpoint to assign a user task.
+    const url = `${ZeebeApiUrl}/user-tasks/${userTaskKey}/assignment`;
+  
+    // Configure the API call.
+    const options = {
+      method: "POST",
+      url,
+      headers: {
+        Accept: "application/json",
+        Authorization: `Bearer ${accessToken}`
+      },
+      data: {
+        // The body contains information about the new assignment.
+        assignee: assignee
+      }
+    };
+  
+    try {
+      // Call the add endpoint.
+      const response = await axios(options);
+  
+      // Process the results from the API call.
+      const assignee = response.data;
+  
+      console.log(
+        `User task assigned to ${assignee.assignee}.`
+      );
```

Comment:
```suggestion
    if (response.status === 204) {
      console.log(`User task assigned to ${assignee}.`);
    } else {
      // Emit an unexpected error message.
      console.error("Unable to assign this user!");
    }
```

1. The indentation is off in this entire function, so this suggestion looks a little funny. Any indentation changes we apply here should be applied to the entire function.
2. This endpoint does not return anything when it succeeds. It gives a 204 response with no body.